### DISCUSSION TITLE
eme_bookings vs eme_attendees

### DISCUSSION THREAD

**Discussion by TimB111** [liedekef/events-made-easy#169] [General]
I was testing eme_bookings and eme_attendees to see the differences and I put both of these shortcodes on a test page. 
[eme_bookings id=1234 rsvp_status=1]
[eme_attendees id=1234 rsvp_status=1]

Several bookings have already been made for the event with id 1234, and eme_bookings gives a list of surnames, but eme_attendees returns "No bookings made yet". Why would it give this?

Also, I'm wondering if I'm confusing these. Does 1 booking have multiple attendees (if someone books multiple places), or is the relationship the other way, that 1 attendee can have multiple bookings?

**Comment by liedekef** [comment]
I just tried it here and the result is as expected. What is the status of your bookings? And your EME version?

Concerning the second question: one attendee can have multiple bookings

**Reply by TimB111** [reply]
That's odd, I've just tried it out using a few different event IDs and eme_attendees is working now. If I manage to recreate it I'll post back.

Thanks for confirming the attendee-bookings relationship.

**Reply by TimB111** [reply]
By the way, my current EME version is 2.3.48. I'll update at the next version.

And the event was live, in the future, and the bookings were all approved and paid.

**Reply by liedekef** [reply]
2.3.48 is already old. We're at 2.3.65 currently
